core/src/main/java/apoc/ApocSettings.java

```diff
@@ -107,6 +107,9 @@ public Class<IPAddressString> getType()
 
     public static final Setting<Long> apoc_jobs_scheduled_num_threads = newBuilder(APOC_CONFIG_JOBS_SCHEDULED_NUM_THREADS, LONG, null).build();
 
+    @Description("maximum decompression ratio when using procedures that will decompress a compressed file passed as parameter")
+    public static final Setting<Integer> apoc_decompression_ratio = newBuilder(APOC_MAX_DECOMPRESSION_RATIO, INT, DEFAULT_DECOMPRESSION_RATIO).build();
+
```

Comment:
If I remove this from the file or even if I add a constraint `.addConstraint(min(1))` before the `.build()` call it has no effect, so I think we are not validating apoc settings at all?